learning-objectives/intl/es.yml

```diff
@@ -2087,3 +2087,276 @@ design-systems/atomic-design:
+
+csharp/data-types/dynamic-types:
+  title: Tipos dinámicos
+  links:
+    - title: dynamic (Referencia de C#) - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/language-reference/builtin-types/reference-types#the-dynamic-type
+    - title: Uso del tipo dynamic - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/advanced-topics/interop/using-type-dynamic
+
+csharp/collections:
+  title: Colecciones
+  links:
+    - title: Colecciones (Referencia de C#) - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/language-reference/builtin-types/collections
+
+csharp/collections/lists:
+  title: Listas
+  links:
+    - title: List<T> Clase - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/api/system.collections.generic.list-1?view=net-8.0
+    - title: Aprenda a administrar colecciones de datos mediante List<T> en C# - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/tour-of-csharp/tutorials/arrays-and-collections
+
+csharp/collections/arrays:
+  title: Arreglos
+  links:
+    - title: Matrices - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/language-reference/builtin-types/arrays
+
+csharp/collections/sets:
+  title: Conjuntos (Sets)
+  links:
+    - title: HashSet<T> Clase - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/api/system.collections.generic.hashset-1?view=net-8.0
+    - title: HashSet in C# with Examples (en inglés)
+      url: https://www.geeksforgeeks.org/hashset-in-c-sharp-with-examples/
+
+csharp/variables:
+  title: Variables
+  links:
+    - title: Variables (en inglés) - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/language-reference/language-specification/variables
+    - title: C# | Variables
+      url: https://www.geeksforgeeks.org/c-sharp-variables/
+    - title: Variables y Tipos en C#
+      url: https://desarrolloweb.com/articulos/variables-tipos-csharp
+
+csharp/control-structures:
+  title: Estructuras de Control
+
+csharp/control-structures/conditionals:
+  title: Condicionales
+  links:
+    - title: if-else (Referencia de C#) - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/language-reference/statements/selection-statements#the-if-statement
+
+    - title: switch (Referencia de C#) - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/language-reference/statements/selection-statements#the-switch-statement
+
+    - title: Operadores de igualdad, prueban si dos objetos son iguales o no - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/language-reference/operators/equality-operators
+
+    - title: Operadores de comparación (referencia de C#) - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/language-reference/operators/comparison-operators
```

Comment:
con la estructura [que defines arriba  ](https://github.com/Laboratoria/curriculum/pull/1747/files#diff-10b256d70ed906d0018e53f579eac3c5f2bdd33ba5509a5041009eb112e6d63bR287)
```
- control-structures:
      - conditionals:
          - if
          - switch
      - loops:
          - for
          - while
          - do-while
          - foreach
  ```
necesitas definir
`csharp/control-structures`
`csharp/control-structures/conditionales`
`csharp/control-structures/conditionales/if`
`csharp/control-structures/conditionales/switch`
`csharp/control-structures/loops`
`csharp/control-structures/loops/for` etc etc

creo mejor solo tienes 
`conditionals` y `loops` [como las otras idiomas](https://github.com/Laboratoria/curriculum/blob/main/learning-objectives/data.yml#L35-L37), y puedes explicar la variedad en el `title` y con los recursos.
Fijate en los otras idiomas, [javascript por ejemplo ](https://github.com/Laboratoria/curriculum/blob/main/learning-objectives/intl/es.yml#L191-L217)

Reply:
Se ha simplificado, tiene más sentido, gracias!